jetpack.php

```diff
@@ -129,6 +129,12 @@ function vip_jetpack_load() {
 		return;
 	}
 
+	if ( defined( 'VIP_IS_FEDRAMP' ) && constant( 'VIP_IS_FEDRAMP' ) ) {
```

Comment:
Thoughts on moving that conditional in the above block? That way we avoid the repetition.